Languages/Tortoise.pot

```diff
@@ -5803,10 +5811,34 @@ msgstr ""
 msgid "Last Modified:"
 msgstr ""
 
+#. Resource IDs: (1580)
+msgid "Last N commit(s)"
+msgstr ""
+
+#. Resource IDs: (1582)
+msgid "Last N month(s)"
+msgstr ""
+
+#. Resource IDs: (1583)
+msgid "Last N week(s)"
+msgstr ""
+
+#. Resource IDs: (1581)
+msgid "Last N year(s)"
+msgstr ""
+
 #. Resource IDs: (1616)
 msgid "Last known &good:"
 msgstr ""
 
+#. Resource IDs: (1578)
+msgid "Last selected date"
+msgstr ""
+
+#. Resource IDs: (1579)
+msgid "Last selected item"
```

Comment:
Agree with "Not clear". :)

I will drop it. or you do it for me?
I am going to sleep. see u. :)